src/shared/sam/cli/samCliBuild.ts

```diff
@@ -86,6 +90,12 @@ export class SamCliBuildInvocation {
         this.addArgumentIf(invokeArgs, !!this.args.useContainer, '--use-container')
         this.addArgumentIf(invokeArgs, !!this.args.skipPullImage, '--skip-pull-image')
         this.addArgumentIf(invokeArgs, !!this.args.manifestPath, '--manifest', this.args.manifestPath!)
+        this.addArgumentIf(
+            invokeArgs,
+            !!this.args.parameterOverrides && this.args.parameterOverrides.length > 0,
+            '--parameter-overrides',
+            ...(this.args.parameterOverrides ?? [])
+        )
```

Comment:
To get basic test coverage, could add a case in `src/test/shared/sam/cli/samCliBuild.test.ts`  (just plop it in the `it('invokes sam build with args')` test).

same for `src/shared/sam/cli/samCliLocalInvoke.ts`